frontend/pages/admin/IntegrationsPage/cards/Certificates/helpers.tsx

```diff
@@ -0,0 +1,79 @@
+import {
+  ICustomScepProxy,
+  IDigicertCertificate,
+  IScepIntegration,
```

Comment:
Seems like we might want to do some renaming of these interfaces here, perhaps:

`ICertificatesIntegrationCustomSCEP`
`ICertificatesIntegrationDigicert`
`ICertificatesIntegrationNDES`

Reply:
those make sense. will make the change.